src/electron-main.js

```diff
@@ -386,50 +386,18 @@ ipcMain.on('ipcCall', async function(ev, payload) {
                 mainWindow.webContents.goForward();
             }
             break;
-        case 'setLanguage': {
-            // work around `setSpellCheckerLanguages` being case-sensitive by converting to expected case
-            const caseMap = {};
-            const availableLanguages = mainWindow.webContents.session.availableSpellCheckerLanguages;
-            availableLanguages.forEach(lang => {
-                caseMap[lang.toLowerCase()] = lang;
-            });
-
-            if (!caseMap["en"]) {
-                // default special-case for `en` as in Riot is actually implies `en-GB`. `en-US` is distinct.
-                // this way if `en` is requested and not available and `en-GB` is available it'll be used.
-                caseMap["en"] = caseMap["en-gb"];
+        case 'setSpellCheckLanguages':
+            if (args[0] && args[0] != []) {
```

Comment:
Is the goal to test whether the length is greater than 0...? If so, the following is a bit more idiomatic:

```suggestion
            if (args[0] && args[0].length > 0) {
```